apis/dkjson

```diff
@@ -11,7 +11,7 @@ David Kolf's JSON module for Lua 5.1/5.2
 This module writes no global values, not even the module table.
 Import it using
 
-    json = require ("dkjson")
+    json = require ("github.rom/apis/dkjson")
```

Comment:
On the require being disabled in CC you are incorrect... on the line being inside of a comment (facepalm).  I didn't have my syntax highlighting set in sublime when editing this file.